DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -138,7 +161,16 @@ export CONFIG_EXTRA_PROCESS_o2_primary_vertexing_workflow="$PVERTEXER;$VDRIFTPAR
 export CONFIG_EXTRA_PROCESS_o2_secondary_vertexing_workflow="$SVTX"
 
 # ad-hoc settings for its-tpc matching
-export ITSTPCMATCH="tpcitsMatch.maxVDriftUncertainty=0.2;tpcitsMatch.safeMarginTimeCorrErr=10.;tpcitsMatch.cutMatchingChi2=1000;tpcitsMatch.crudeAbsDiffCut[0]=5;tpcitsMatch.crudeAbsDiffCut[1]=5;tpcitsMatch.crudeAbsDiffCut[2]=0.3;tpcitsMatch.crudeAbsDiffCut[3]=0.3;tpcitsMatch.crudeAbsDiffCut[4]=10;tpcitsMatch.crudeNSigma2Cut[0]=200;tpcitsMatch.crudeNSigma2Cut[1]=200;tpcitsMatch.crudeNSigma2Cut[2]=200;tpcitsMatch.crudeNSigma2Cut[3]=200;tpcitsMatch.crudeNSigma2Cut[4]=900;"
+[[ -z TPCITSTIMEBIAS ]] && TPCITSTIMEBIAS=0
+[[ -z TPCITSTIMEERR ]] && TPCITSTIMEERR=0
+if [[ $PERIOD == "LHC22s" ]]; then
+  MAX_VDRIFT_UNC=0.04
+  CUT_MATCH_CHI2=150
+else
+  MAX_VDRIFT_UNC=0.2
+  CUT_MATCH_CHI2=1000
+fi
```

Comment:
I would apply chi2=150 to all production, as I commented in the jira. The maxVDriftUncertainty is relevant only when VD calibration is on and 0.04 should be enough.

Reply:
Ok, I had not understood from JIRA, sorry. 
VD calibration should be always on, so I changed to 0.04 as default.

Reply:
OK to merge, but what do you mean by VD calibration should be always on? I was referring to VDriftTgl calibration workflow, maxVDriftUncertainty acts only if the input for this calibration is requested (`o2-tpcits-match-workflow --produce-calibration-data`)

Reply:
I thought you meant to have the calibration for the runs during the processing, from CCDB, which I found a bit unexpected as a comment. 
This is a script for async reco, here we never run the vdrift calibration, we just use it. Then maybe it is better to remove this?

Reply:
As you prefer, it is anyway ignored unless the VDrift calibration workflow is enabled.

Reply:
Ok then i will merge now, since i am not at the laptop, but i will remove it at the next occasion.

thanks!